corehq/apps/app_manager/views/modules.py

```diff
@@ -1078,7 +1078,13 @@ def _save_case_list_lookup_params(short, case_list_lookup, lang):
 
 @require_GET
 @require_deploy_apps
-def view_module(request, domain, app_id, module_id):
+def view_module(request, domain, app_id, module_unique_id):
+    from corehq.apps.app_manager.views.view_generic import view_generic
+    return view_generic(request, domain, app_id, module_unique_id=module_unique_id)
+
+@require_GET
+@require_deploy_apps
+def view_module_legacy(request, domain, app_id, module_id):
```

Comment:
Good call keeping these views. Might be worth a comment explaining why, don't think it'll be obvious to a new dev.